internal/dtr/in_test.go

```diff
@@ -59,4 +59,23 @@ var _ = Describe("In", func() {
 			}))
 		})
 	})
+
+	Context("empty/no-op configuration", func() {
+		It("should not fail", func() {
+			_, err := In(feed(Config{}))
```

Comment:
Should it fail with an empty configuration? Empty as in, there is no run command?

Reply:
No it shouldn't.
Reason:

One could just use the resource only for Check as trigger for pipelines. But they will probably not need the In.

If we would fail in case there is no Run in In defined they couldn't use this resource anymore. That's why it should not fail

Reply:
I've now added an explanation here:
https://github.com/homeport/duct-tape-resource/pull/33/files#diff-d2789d50d817e2eb2d8d2c1d55bd09d9cfff09af6e4f612b52e9816748ab4505R115-R120